lib/pages/lists.dart

```diff
@@ -1,10 +1,69 @@
 import 'package:MobileOne/localization/localization.dart';
+import 'package:MobileOne/services/user_service.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:MobileOne/pages/widget_list.dart';
 
-class Lists extends StatelessWidget {
-  // This widget is the root of your application.
+const Color ORANGE = Colors.deepOrange;
+
+class Lists extends StatefulWidget {
+  State<StatefulWidget> createState() {
+    return ListsState();
+  }
+}
+
+String defaultImage = "assets/images/basket_my_lists.png";
+String numberOfItemShared = "3 partages";
+
+class ListsState extends State<Lists> {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(body: Center(child: Text(getString(context, "my_lists"))));
+    return StreamBuilder<DocumentSnapshot>(
+      stream: Firestore.instance
+          .collection("owners")
+          .document(UserService().user.uid)
+          .get()
+          .asStream(),
+      builder: (context, snapshot) {
+        if (!snapshot.hasData) {
+          return CircularProgressIndicator();
+        }
+        var wishlist = [];
+        if (snapshot.data.data["lists"] != null) {
```

Comment:
snapshot.data.data may be null on first connection 

you can replace by 

var wishlist = snapshot?.data?.data?.get("lists") ?? [];